web/app.js

```diff
@@ -1256,6 +1268,15 @@ let PDFViewerApplication = {
       // Setting the default one.
       this.pdfViewer.currentScaleValue = DEFAULT_SCALE_VALUE;
     }
+
+    // These are undefined if setInitialValues is called without a second
```

Comment:
I think you mean `setInitialView` here. Moreover, I think you can get rid of this if you change the checks below to `if (Number.isInteger(scrollMode)) {` and equivalent for `spreadMode`.